lib/injekt.rb

```diff
@@ -0,0 +1,29 @@
+class Array
+
+  def injekt(*args, &block)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for injekt is too high. [23.54/15]